service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceTest.java

```diff
@@ -120,14 +121,6 @@ public class ControlledResourceServiceTest extends BaseConnectedTest {
   /** The default GCP location to create notebooks for this test. */
   private static final String DEFAULT_NOTEBOOK_LOCATION = "us-east1-b";
 
-  private static final String DEST_DATASET_NAME = TestUtils.appendRandomNumber("dest_dataset_name");
```

Comment:
In this file, I delete many did not used functions and variables, also remove some did not used exceptions to help the file clean and efficient.